config/index.ts

```diff
@@ -21,6 +21,7 @@ export const config = {
   uploadTo: 'aws',
   name: process.env.NODE_ENV,
   authBaseUrl: process.env.AUTHBASE_URL,
+  basicAuthUser: process.env.BASIC_AUTH_USER,
```

Comment:
we probably want to delete that as we've hard-coded the expected auth user for HID in `hpc-api-core` if i remember correctly.

If not, that's something we want to do, and also this change doesn't belong in this commit ;)

Reply:
You are right on both points :)